packages/components/src/FormatRelativeDateTime/FormatRelativeDateTime.mdx

```diff
@@ -0,0 +1,46 @@
+---
+name: FormatRelativeDateTime
+menu: Components
+route: /components/format-relative-date-time
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { CivilDate } from "@std-proposal/temporal";
+import { FormatRelativeDateTime } from ".";
+
+# FormatRelativeDateTime
+
+<ComponentStatus stage="pre" responsive="yes" accessible="yes" />
+
+A FormatRelateDateTime displays the date and time using relative wording or
+approximate values.
+
+Rules: If the message was received less than an hour ago the format is x minutes
+ago
+
+If the message was received today, but more than an hour ago, the format becomes
+the time it was received
+
+If the message was received this within the last six days, the format is Sun,
+Mon, Tue, Wed, Thu, Fri, Sat
+
+If the message was received between six days ago and yesterday’s date one year
+previous, the format is Month Date
+
+If the message was received exactly one year ago or more than one year ago, the
+format is Month Day, Year
```

Comment:
I think this whole rules content should have its own section below the `Simple Example`. 

```
## Rules

...
```

Reply:
Done